CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ found [here](packages/broker/CHANGELOG.md).
 
 #### Added
 
+- Add flag `--raw` (or shorthand `-r`) to command `stream subscribe` for raw subscription (https://github.com/streamr-dev/network/pull/1317)
```

Comment:
Mayve changelog items could be as short as possible? I.e. not mentioning the shorthands?